attestation/git/git.go

```diff
@@ -85,7 +112,86 @@ func (a *Attestor) Attest(ctx *attestation.AttestationContext) error {
 		return err
 	}
 
+	commit, err := repo.CommitObject(head.Hash())
+	if err != nil {
+		return err
+	}
+
+	a.CommitDigest = cryptoutil.DigestSet{
+		{
+			Hash:   crypto.SHA1,
+			GitOID: false,
+		}: commit.Hash.String(),
```

Comment:
I'm not sure that a digest set makes sense used like this. The resulting data in this case is going to look something like

```
"commitdigest" {
  "63ff2342b54bd1ffc9dbdf527130e7dcab51d8d1": {
    "sha1": "63ff2342b54bd1ffc9dbdf527130e7dcab51d8d1"
  }
}
```

This should probably just look like

```
"commitdigest": {
  "sha1": "63ff2342b54bd1ffc9dbdf527130e7dcab51d8d1"
}
```